Update fit_gev

stellema opened the pull request.

stellema committed:
Add linear trend test for non stationary gev

stellema committed:
Update eva.py

DamienIrving commented on unseen/eva.py:
```diff
@@ -247,18 +254,18 @@ def _fit(
                     theta = fit(data, **kwargs)
                 else:
+                    # Return NaNs
+                    theta = theta * np.nan
```
That's a neat solution :smile:

stellema committed:
Update eva.py

Add non-stationary relative fit tests to fit_gev. The default is set to skip these tests because I'm not sure if the test data is trended enough to pass the any of the tests.

stellema committed:
Add GEV relative fit tests

Delete non-stationary part of "check_gev_fit", add "check_relative_fit" to fit_gev and update eva tests
